js/src/components/paid-ads/budget-section/index.js

```diff
@@ -49,6 +76,8 @@ const BudgetSection = ( props ) => {
 								) }
 								suffix={ currency }
 								{ ...getInputProps( 'amount' ) }
+								value={ amountValue }
+								readOnly={ disabled }
```

Comment:
❔  I think this should be `disabled` instead of `readonly`?

```suggestion
								disabled={ disabled }
```

Here's my testing video with the current `readOnly={ disabled }` code:

https://user-images.githubusercontent.com/417342/155371525-cea36590-24c6-4268-b8ba-bbc3d7004515.mp4

Reply:
In order to get as close to the visual design while not overriding the style of wp-components, using `readOnly` is a compromise.

I added another workaround style to avoid the focused style though it still has to override them 😅 , I think it'd be better than overriding the disabled style.

Reply:
> I [added another workaround style](https://github.com/woocommerce/google-listings-and-ads/pull/1268/commits/17c766ab83955f84344a34b36f80398f7f1fba88) to avoid the focused style [...]

I [committed another one](https://github.com/woocommerce/google-listings-and-ads/pull/1268/commits/2b07743f4242b90da99b263ce67c970218b2fd51) to fix that focused style is still appears when clicking on its `<label>`. It should be ok now 😅 

![Kapture 2022-02-24 at 14 44 08](https://user-images.githubusercontent.com/17420811/155472443-2c42e9a3-53f9-4ca6-8b96-b143e575dde3.gif)

Reply:
> In order to get as close to the visual design while not overriding the style of wp-components, using `readOnly` is a compromise.

💅  Personally sometimes I would use my own best judgement, and in this case, I would just use `disabled` to achieve the goal of making it not editable, even though it may be slightly off in visual design. I get your point here though, and I'm fine with that, and thank you for your desire on getting close to visual design. 🙂  👍